data/src/main/java/com/ivyclub/data/repository/ContactRepositoryImpl.kt

```diff
@@ -16,4 +21,26 @@ class ContactRepositoryImpl @Inject constructor(
     override fun saveFriend(friendData: FriendData) {
         contactDAO.insertFriendData(friendData)
     }
+
+    override fun setShowOnBoardingState(state: Boolean) {
+        myPreference.setShowOnBoardingState(state)
+    }
+
+    override fun getShowOnBoardingState(): Boolean {
+        return myPreference.getShowOnBoardingState()
+    }
+
+    override fun setNotificationTime(start: String, end: String) {
+        myPreference.setNotificationTime(NOTIFICATION_START, start)
+        myPreference.setNotificationTime(NOTIFICATION_END, end)
+    }
+
+    override fun setNotificationOnOff(state: Boolean) {
+        myPreference.setNotificationOnOff(state)
+    }
+
+    companion object {
+        const val NOTIFICATION_START = "NOTIFICATION_START"
+        const val NOTIFICATION_END = "NOTIFICATION_END"
+    }
```

Comment:
```suggestion
    companion object {
        private const val NOTIFICATION_START = "NOTIFICATION_START"
        private const val NOTIFICATION_END = "NOTIFICATION_END"
    }
```
내부에서만 사용되는 거면 접근 제한자 추가해줘도 될 것 같습니다